### PR TITLE
Plot TFR x-axis in seconds instead of milliseconds

### PR DESCRIPTION
**Pull request by cbrnr**
Change time axis scale to seconds in `AverageTFT.plot` and `AverageTFR.plot_topo`.

### CODE REVIEW

**Comment by agramfort** [comment]
hum evoked.plot and many other methods use ms

did I miss a conversation?

**Comment by cbrnr** [comment]
Here: https://github.com/mne-tools/mne-python/pull/4566#issuecomment-329785452

File: mne/time_frequency/tfr.py [hunk 1]
```diff
@@ -1151,7 +1151,7 @@ def plot(self, picks, baseline=None, mode='mean', tmin=None, tmax=None,
                                         mode=mode, layout=layout)
             _imshow_tfr(ax, 0, tmin, tmax, vmin, vmax, onselect_callback,
                         ylim=None, tfr=data[idx: idx + 1], freq=freqs,
-                        x_label='Time (ms)', y_label='Frequency (Hz)',
+                        x_label='t (s)', y_label='f (Hz)',
```
Comment (agramfort) [inline mne/time_frequency/tfr.py]: t and f don't seem explicit enough for me

Comment (cbrnr) [reply mne/time_frequency/tfr.py]: When someone computes TFRs it should be clear what t (s) and f (Hz) mean. I prefer shorter labels to de-clutter the figures. But of course labels can be changed with MPL afterwards, although this is more work. Should I change it back?

**Comment by cbrnr** [comment]
Any ideas what's wrong with CircleCI?

File: mne/time_frequency/tfr.py (hunk 1 above)
Comment (jona-sassenhagen) [reply mne/time_frequency/tfr.py]: I don't think one-character labels are a good idea. You know your plot is supposed to be a TFR map, but when you're showing it to somebody else, the label is what tells them it's a TFR plot.

**Comment by larsoner** [comment]
> hum evoked.plot and many other methods use ms

evoked.plot is the only other one AFAIK, seconds is much more common. We
should consider changing it, too.

**Comment by jona-sassenhagen** [comment]
See also https://github.com/mne-tools/mne-python/issues/3164

**Comment by agramfort** [comment]
-100 on shorter names. You should not put in papers "t (s)" this is cryptic.

maybe we can swtich between ms and s automatically. If more than 1.5s long then switch to seconds ortherwise use ms. If you look at early ERPs/ERFs you prefer to see ms

**Comment by jona-sassenhagen** [comment]
Comedy option: if instance.times is in integers, it's ms, if it's in floats, it's seconds.

**Comment by agramfort** [comment]
@jona-sassenhagen you're scaring me...

**Comment by jona-sassenhagen** [comment]
:D

**Comment by cbrnr** [comment]
I don't know, t and f are such standard symbols that everyone should be able to identify (even more so if they come in combination with theirs units). Anyway, I can revert this change since no one will give me +100 :smile:.

Regarding the unit of time, let's always use seconds. I know it's very common in psychology to use ms, but for most cases the standard unit s is perfectly fine.

**Comment by cbrnr** [comment]
`git grep -E "(T|t)ime \(ms\)"`:

```
doc/manual/gui/browse.rst:**Starting time (ms)**
doc/manual/gui/browse.rst:**Ending time (ms)**
examples/decoding/plot_ems_filtering.py:plt.xlabel('Time (ms)')
examples/decoding/plot_ems_filtering.py:plt.xlabel('Time (ms)')
examples/inverse/plot_compute_mne_inverse_epochs_in_label.py:plt.xlabel('time (ms)')
examples/inverse/plot_compute_mne_inverse_epochs_in_label.py:plt.xlabel('time (ms)')
examples/inverse/plot_compute_mne_inverse_raw_in_label.py:plt.xlabel('time (ms)')
examples/inverse/plot_covariance_whitening_dspm.py:    ax_dynamics.set_xlabel('Time (ms)')
examples/inverse/plot_dics_beamformer.py:plt.xlabel('Time (ms)')
examples/inverse/plot_label_source_activations.py:plt.xlabel('Time (ms)')
examples/inverse/plot_lcmv_beamformer.py:plt.xlabel('Time (ms)')
examples/inverse/plot_lcmv_beamformer_volume.py:plt.xlabel('Time (ms)')
examples/inverse/plot_mixed_source_space_inverse.py:axes.set(xlabel='Time (ms)', ylabel='MNE current (nAm)')
examples/inverse/plot_morph_data.py:plt.xlabel('time (ms)')
examples/inverse/plot_read_stc.py:plt.xlabel('time (ms)')
examples/stats/plot_cluster_stats_evoked.py:plt.xlabel("time (ms)")
examples/stats/plot_fdr_stats_evoked.py:plt.xlabel("Time (ms)")
examples/time_frequency/plot_source_space_time_frequency.py:plt.xlabel('Time (ms)')
examples/time_frequency/plot_time_frequency_simulated.py:ax.set(ylim=freqs[[0, -1]], xlabel='Time (ms)')
mne/viz/_3d.py:    ax.set_xlabel('Time (ms)', fontsize=18)
mne/viz/evoked.py:            ax.set_xlabel('time (ms)')
mne/viz/evoked.py:    ax.set_xlabel('time (ms)')
mne/viz/ica.py:    ax.set_xlabel('Time (ms)')
mne/viz/ica.py:    ax2.set_xlabel('time (ms)')
mne/viz/topomap.py:    ax.set_xlabel('Time (ms)')
tutorials/plot_mne_dspm_source_localization.py:plt.xlabel('time (ms)')
tutorials/plot_stats_cluster_1samp_test_time_frequency.py:plt.xlabel('Time (ms)')
tutorials/plot_stats_cluster_spatio_temporal_repeated_measures_anova.py:plt.xlabel('Time (ms)')
tutorials/plot_stats_cluster_time_frequency.py:plt.xlabel('Time (ms)')
tutorials/plot_stats_cluster_time_frequency_repeated_measures_anova.py:    plt.xlabel('Time (ms)')
tutorials/plot_stats_cluster_time_frequency_repeated_measures_anova.py:plt.xlabel('Time (ms)')
tutorials/plot_stats_cluster_time_frequency_repeated_measures_anova.py:plt.xlabel('Time (ms)')
```

**Comment by jona-sassenhagen** [comment]
If you get around to doing that, please also capitalise everything, see #4534

**Comment by cbrnr** [comment]
@jona-sassenhagen 😱 and I just wanted to propose to lowercase everything! Well I guess this is a matter of style, but MNE should at least be consistent.

**Comment by jona-sassenhagen** [comment]
Well, Alex said we should discuss what's better either way. But as @bgagl points out in #4535 , e.g. Nature requires:

>Lettering in figures (labelling of axes and so on) should be in lower-case type, with the first letter capitalized and no full stop.)

And I'm sure you wanna publish in Nature :)

**Comment by cbrnr** [comment]
Fine, you win :smile:. So can we leave it at that in this PR? Changing everything to seconds instead of ms should probably be tackled in another PR. Same for capitalizing the labels.

**Comment by larsoner** [comment]
Yes let's update other functions separately

**Comment by cbrnr** [comment]
Where's CircleCI?

**Comment by larsoner** [comment]
Some random GitHub failure, I restarted it

**Comment by agramfort** [comment]
thx @cbrnr

**Pull request merged**